### PR TITLE
(bug)table: results overlap with search bar

### PR DESCRIPTION
**Pull request by friedaxvictoria**
Now it reacts similarly to the menu button. The search results are only displayed at a certain size. If the text is put above the icon, it looks a little bit misshapen and it also doesn't save much space. Should you still want to keep the search results, we'll have to think of something else.

### CODE REVIEW

**Comment by DanielHabenicht** [comment]
Mhm, maybe we should move the search bar to the left? 
There is so much unused space around it and on bigger screens it still overlaps.
image

**Comment by friedaxvictoria** [comment]
Obviously, that would be the easiest option. I don't know whether that would still look as good. @T-Systems-MMS/phonebook-developers ?

**Comment by friedaxvictoria** [comment]
2020-05-11_08h27_26

For now it is the same width as before. But if the searchbar is on the left, it might be visually nicer to always have it at a 100%. I'm just not sure whether it is to overpowering...

Also, the title and the searchbar are aligned on the left. However, this is not the case for the rest of the table. Should they all align on the left?

**Comment by DanielHabenicht** [comment]
> For now it is the same width as before. But if the searchbar is on the left, it might be visually nicer to always have it at a 100%. I'm just not sure whether it is to overpowering...

I think thats okay, how about @T-Systems-MMS/phonebook-developers ?

> Also, the title and the searchbar are aligned on the left. However, this is not the case for the rest of the table. Should they all align on the left?

What do you mean by that?

**Comment by friedaxvictoria** [comment]
2020-05-12_07h17_22

The difference is minimal but the title, the searchbar, and the searched for people don't start in the same place.

File: Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html [hunk 1]
```diff
@@ -239,9 +239,13 @@
           </mat-toolbar-row>
           <mat-toolbar-row>
             <div class="row-toolbar">
-              <app-search id="search"></app-search>
+              <app-search id="search"> </app-search>
+              <div class="pb-spacer"></div>
               <div class="table-settings" *ngIf="displayTableSettings">
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html:244]: Wird mir gar nicht angezeigt...

Comment (friedaxvictoria) [reply Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html:244]: Ja, das war wichtig bevor es 100% Breite hatte. Ich entferne es...

Comment (DanielHabenicht) [reply Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html:244]: Also ich meine mir werden weder Einstellungen noch Anzahl der Ergebnisse angezeigt.

**Comment by DanielHabenicht** [comment]
image
Still there is no settings button and result count

**Comment by friedaxvictoria** [comment]
The settings button and result count only show up when the user is using the search bar. Otherwise, it is hidden. You added a property which returns true whenever there is 'search' in the router url. The button and result count were ngIf bound to that property.

**Comment by DanielHabenicht** [comment]
ok yeah my bad 🙈

**Comment by DanielHabenicht** [comment]
@friedaxvictoria is it working for you in the main demo?

**Comment by friedaxvictoria** [comment]
As I mentioned before, I can’t seem to get localhost to work on my personal laptop. There isn’t a „secure connection to the server“. I don’t know whether I need proxys...

**Pull request merged**